Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/GameWrapper.java

```diff
@@ -18,39 +15,46 @@
 import nl.soccar.physics.GameEngine;
 import nl.soccar.physics.models.BallPhysics;
 import nl.soccar.physics.models.CarPhysics;
+import nl.soccar.physics.models.ObstaclePhysics;
+import nl.soccar.socnet.connection.Connection;
 import nl.soccar.socnet.message.Message;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Timer;
 import java.util.TimerTask;
+import java.util.stream.Collectors;
 
 /**
  * @author PTS34A
  */
 public final class GameWrapper {
 
+    private final List<PlayerWrapper> playersReady = new ArrayList<>();
+
     private final SessionWrapper session;
     private final Game game;
 
     private final GameEngine engine;
-    private final Timer timer;
+    private Timer timer;
 
     public GameWrapper(SessionWrapper session, Game game) {
         this.session = session;
         this.game = game;
 
         engine = new GameEngine(game);
-        timer = new Timer();
     }
 
-    public void start() {
+    public void requestStart() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)